nxrm-aws-ha-helm/values.yaml

```diff
@@ -15,6 +15,8 @@ statefulset:
   clusterName: nxrm-nexus
   logsRegion: us-east-1
   fluentBitVersion: 2.28.0
+  skipDefaultRepositories: true
+  provisionDefaultBlobStore: false
```

Comment:
These shouldn't be required:

https://github.com/sonatype/nexus-internal/blob/main/components/nexus-repository-config/src/main/java/org/sonatype/nexus/repository/internal/blobstore/BlobStoreManagerImpl.java#L136-L143

https://github.com/sonatype/nexus-internal/blob/main/components/nexus-repository-services/src/main/java/org/sonatype/nexus/repository/manager/internal/RepositoryManagerImpl.java#L239

Reply:
Reverted